src/components/Calculator.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+class Calculator extends React.PureComponent {
+  render() {
+    return (
+      <div className="calculator">
+        <input type="number" className="calc-input" />
+        <table className="calc-buttons">
+          <tr className="tr">
+            <td className="button1">AC</td>
+            <td className="button1">+/-</td>
+            <td className="button1">%</td>
+            <td className="button2">+</td>
```

Comment:
- [ ] From the design this should be the divide sign (&#247;), not +. Kindly fix that. If you do not have the  sign on your keyboard, consider using this HTML entity code `&#247;`